spec/bolt/cli_spec.rb

```diff
@@ -2302,14 +2311,71 @@ def stub_config(file_content = {})
         end
       end
 
-      it 'init creates a new project in the current working directory' do
+      it 'creates a new project in the current working directory' do
         Dir.mktmpdir do |dir|
           file = File.join(dir, 'bolt.yaml')
           cli = Bolt::CLI.new(%w[project init])
           Dir.chdir(dir) { cli.execute(cli.parse) }
           expect(File.file?(file)).to be
         end
       end
+
+      it 'warns when a bolt.yaml already exists' do
+        Dir.mktmpdir do |dir|
+          config = File.join(dir, 'bolt.yaml')
+          cli    = Bolt::CLI.new(%W[project init #{dir}])
+
+          FileUtils.touch(config)
+          cli.execute(cli.parse)
+
+          expect(@log_output.readlines).to include(/Found existing project directory at #{dir}/)
+        end
+      end
+
+      context 'with modules' do
+        it 'creates a Puppetfile and installs modules with dependencies' do
+          Dir.mktmpdir do |dir|
+            puppetfile = File.join(dir, 'Puppetfile')
+            modulepath = File.join(dir, 'modules')
+
+            cli = Bolt::CLI.new(%W[project init #{dir} --modules puppetlabs-apt])
```

Comment:
If you're worried about test times, `puppetlabs-stdlib` is one of the worst modules due to the sheer number of releases, so it can take some time query the Forge.

Reply:
Good to know. I'll switch that out with another one.